src/views/tutorial-view/components/next-previous/components/directional-link-box/directional-link-box.module.css

```diff
@@ -1,101 +1,36 @@
-.card {
-  border: 1px solid magenta; /** @TODO style these to spec, remove any prev imp styles necessary - https://www.figma.com/file/VD7ahvXuXWJApeGnhbW4hv/Dev-Portal?node-id=6298%3A73041 */
-  position: relative;
-  display: block;
-  background: var(--white);
-  border-radius: 5px;
-  margin: 0;
-  padding: 20px;
-}
-
-/* visually hidden */
-.hidden {
-  clip: rect(0 0 0 0);
-  clip-path: inset(50%);
-  height: 1px;
-  overflow: hidden;
-  position: absolute;
-  white-space: nowrap;
-  width: 1px;
-}
-
 .linkbox {
-  composes: card;
   display: flex;
+  flex-wrap: wrap;
   align-items: center;
-  width: 100%;
-  background-color: var(--white);
-  color: var(--black);
-  box-shadow: var(--token-surface-mid-box-shadow);
-  transition: box-shadow 0.3s;
-
-  &[data-direction='previous'] {
-    text-align: left;
-    justify-content: flex-start;
-    &:hover {
-      box-shadow: var(--token-surface-high-box-shadow);
-    }
-  }
-
-  &[data-direction='next'],
-  &[data-direction='final'] {
-    text-align: right;
-    justify-content: flex-end;
-
-    &:hover {
-      box-shadow: var(--token-surface-high-box-shadow);
+  text-align: center;
+  justify-content: center;
+  height: 100%;
+
+  @media (--medium-up) {
+    &.direction-previous {
+      text-align: left;
+      justify-content: flex-start;
     }
-  }
 
-  @media (--small) {
-    &[data-direction='previous'] {
-      justify-content: center;
-    }
-
-    &[data-direction='next'],
-    &[data-direction='final'] {
-      justify-content: center;
+    &.direction-next,
+    &.direction-final {
+      text-align: right;
+      justify-content: flex-end;
     }
   }
 }
 
 .icon {
-  display: flex;
-  align-items: center;
-  flex-shrink: 0;
-  color: var(--black);
-
-  & > div {
-    display: flex;
-  }
+  display: block;
 
-  &[data-direction='previous'] {
+  &.direction-previous {
     order: 0;
```

Comment:
Just noticed these, are the `order` properties actually needed?

Reply:
Great call, the `order: 0` isn't, so I've removed it 🔪 I think it was there just to signal that `.direction-next` does need the `order` prop - it moves the icon after the text in the `flex` flow (removing the need for different markup if the icon is meant to be shown before or after the text).

Reply:
Okay great! Thank you 😊